controllers/nonadmin_controller.go

```diff
@@ -0,0 +1,192 @@
+package controllers
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/go-logr/logr"
+	"golang.org/x/exp/maps"
+	appsv1 "k8s.io/api/apps/v1"
+	corev1 "k8s.io/api/core/v1"
+	k8serror "k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/types"
+	"k8s.io/utils/pointer"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/controller-runtime/pkg/controller/controllerutil"
+
+	oadpv1alpha1 "github.com/openshift/oadp-operator/api/v1alpha1"
+	"github.com/openshift/oadp-operator/pkg/common"
+)
+
+const (
+	nonAdminPrefix            = "non-admin-"
+	controllerManager         = "controller-manager"
+	containerName             = nonAdminPrefix + "manager"
+	nonAdminControllerManager = common.OADPOperatorPrefix + nonAdminPrefix + controllerManager
+	controlPlaneKey           = "control-plane"
+)
+
+var (
+	controlPlaneLabel = map[string]string{
+		controlPlaneKey: nonAdminPrefix + controllerManager,
+	}
+	deploymentLabels = map[string]string{
+		"app.kubernetes.io/component":  "manager",
+		"app.kubernetes.io/created-by": common.OADPOperator,
+		"app.kubernetes.io/instance":   nonAdminPrefix + controllerManager,
+		"app.kubernetes.io/managed-by": "kustomize",
```

Comment:
I think this should be `oadp-operator` instead of `kustomize`

Reply:
this is generated by kubebuilder 
- https://github.com/migtools/oadp-non-admin/blob/master/config/manager/manager.yaml#L26
- https://github.com/mateusoliveira43/nac-operator/blob/main/config/manager/manager.yaml#L26 (example with operator-sdk)

the labels that should reference `oadp-operator` are `app.kubernetes.io/created-by` and `app.kubernetes.io/part-of` https://github.com/migtools/oadp-non-admin/blob/master/config/manager/manager.yaml#L24-L25

do you agree?

Reply:
okay cool